src/Symfony/Component/Routing/Loader/AnnotationDirectoryLoader.php

```diff
@@ -76,6 +76,15 @@ function (\SplFileInfo $current) {
      */
     public function supports($resource, $type = null)
     {
+        if ($type && 'annotation' !== $type) {
+            return false;
+        }
+
+        // classes are handled by the AnnotationClassLoader
+        if (strpos($resource, '\\') || class_exists($resource)) {
```

Comment:
`false !== strpos` to accurately mirror what's done in AnnotationClassLoader?

Reply:
of course